src/pages/operations/fleet.mdx

```diff
@@ -7,32 +7,11 @@ import TurkishAirlinesCards from '../../components/Operations/Fleet/TurkishAirli
 <div class="hero shadow--lw">
   <div class="container">
     <h1 class="hero__title">Fleet</h1>
-    <p class="hero__subtitle">Flying or somethin</p>
+    <p class="hero__subtitle">The Turkish Virtual fleet consists of a range of aircraft that includes the Airbus A321-200, Boeing B737-900, Airbus A330-300, Boeing B777-300, Boeing 787-9, and the Airbus A350-900.
```

Comment:
The hero subtitle is too long. It should either be shortened, or this can be moved to the body text (basically below the hero div as plain markdown)